Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -100,6 +104,16 @@ def get_issue_info(cls, args, repository, commit_objects, commit_bugs, **kwargs)
 
         issue = Tracker.from_string(args.issue)
         commit_bugs_list = list(commit_bugs)
+        if not issue.title:
+            for regex in cls.RES_RADAR:
```

Comment:
Can't we just check `issue.tracker`? Our `Tracker` objects should probably have a `name` variable (they don't right now), if they did, you could just have this be:
```
if not issue.title:
    sys.stderr.write('Could not fetch {} from link. Please verify that the issue exists.\n'.format(issue.tracker.name))
    return None
```